### ISSUE TITLE
Extract local and global representation using finetune model

### ISSUE THREAD

**Issue by namanm04** [nadavbra/protein_bert#84]
how one might extract both local and global representations using finetune model from protein sequence data, considering only the sequence information and labels only, i have generate local and global representations using pretrained model but unable to do for finetune model. my task is protein classification.

**Comment by nadavbra** [comment]
@namanm04 It's basically the exact same code, just run for the fine-tuned rather than base model.
